pytab/core.py

```diff
@@ -372,15 +405,15 @@ def cli(
                 if command["type"] in supported_types:
                     click.echo(f"> > > Current Device: {command['type']}")
                     arguments = command["args"]
-                    arguments = arguments.format(video_file=current_file, gpu=0)
+                    arguments = arguments.format(video_file=current_file, gpu=gpu_ind)
                     test_cmd = f"{ffmpeg_binary} {arguments}"
 
                     valid, runs, result = benchmark(test_cmd)
 
                     test_data["id"] = test["id"]
                     test_data["type"] = command["type"]
                     if command["type"] != "cpu":
-                        test_data["selected_gpu"] = 1
+                        test_data["selected_gpu"] = gpu_id
```

Comment:
Here we leave one value higher then the index (idx) in our system info. This is because the backend server expects Indexing to start on 1, while we index at 0.

Therefore this could be archieved through `gpu_idx +1` too.